packages/connect-voting-disputable/src/__test__/herlpers/numbers.test.ts

```diff
@@ -0,0 +1,49 @@
+import { bn, formatBn } from '../../helpers'
```

Comment:
Nitpick: the directory (`herlpers`—both kind of funny but scary when I initially read it as `herpies`) is typoed.